src/frontend-admin/src/components/reusables/HorizontalBarChart/HorizontalBarChart.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {HorizontalBarChart} from './HorizontalBarChart';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<HorizontalBarChart title={"Test"} amount={8} outOf={10} onClick={() => {}} />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+it('renders without crashing when amount > outOf', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<HorizontalBarChart title={"Test"} amount={8} outOf={4} onClick={() => {}}/>, div);
+  ReactDOM.unmountComponentAtNode(div);
+})
```

Comment:
Awesome! Love to see tests! Note that you can easily reuse props by doing something like 
```
const props = {
amount: 8,
outOf: 4,
}
```
and then using them in your tests like `<HorizontalBarChart {...props} />` can save a lot of duplicated code when you have a lot of tests.